FAQs/How to Model Alerts in OCSF.md

```diff
@@ -0,0 +1,32 @@
+# How to Model Alerts with OCSF - DRAFT
+
+In OCSF, an alertable signal is an event with the `is_alert` attribute set to `true`.  This is a newer attribute, starting with OCSF version 1.4.  The intent of `is_alert = true` is to signal that the event may require immediate attention by its consumer, usually a SIEM system or the product’s management console where an analyst can be notified, tickets created or the events prioritized.
+
+Not all OCSF events are alertable signals, and as of this writing only the `Detection Finding`, `Data Security Finding` event classes and the `Security Control` profile carry the `is_alert` boolean attribute.  Note the `Security Control` profile may be applied to many activity oriented classes, as well as the two aforementioned `Finding` classes, hence when applied any instances of these classes can be alertable signals.
+
+Examples of alertable signals are events with high severity, elevated risk scores, detected malware, MITRE ATT&CK annotations, and rule or policy violation events.  Other alertable signals might be the creation of a Finding based on some type of analysis.
```

Comment:
This is one of the points that makes me feel the `is_alert` is a redundancy. OCSF already has several other methods of describing something as an alert (such as an elevated severity etc)

Reply:
If there is consensus, and we can alleviate confusion, we can drop the `is_alert` flag, but more important is not to shoehorn every alert into `Detection Finding`; that is what started the discussion in the first place . `Security Control` profile based events which have been there since v1.0 are also often alerts.  Also, some alerts are not always high severity, e.g. when the control point blocks the threat.

Reply:
I have added some additional wording to address other ways events can be interpreted as alertable signals, with some of the language below.  There are a number of members who have expressed the need or desire for this more explicit way of expressing alerts.